packages/vscode-extension/src/debug/officeTaskHandler.ts

```diff
@@ -0,0 +1,191 @@
+import { ok } from "@microsoft/teamsfx-api";
```

Comment:
Copyright header.
And also adjust the code owner in `.github/CODEOWNERS`

Reply:
Thanks for your comments. Updated in the latest iteration. Would you please help review it again? Thanks for your effort.